var/spack/repos/builtin/packages/py-jmp/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyJmp(PythonPackage):
+    """JMP is a Mixed Precision library for JAX."""
+
+    homepage = "https://github.com/deepmind/jmp"
+    url = "https://github.com/deepmind/jmp/archive/refs/tags/v0.0.2.tar.gz"
+
+    version('0.0.2', sha256='4d242fb14502b15a7c072e112bdcd7cb5d8b373d9733162eea23e0b9b7dbb6d0')
+    depends_on('py-numpy@1.19.5:', type=('build', 'run'))
+    depends_on('py-dataclasses', type=('build', 'run'), when='^python@3.6.00:3.6')
+    depends_on('py-jax', type=('build', 'run'))
+    depends_on('py-jaxlib', type=('build', 'run'))
```

Comment:
Where do you see these deps listed?

Reply:
again, pulling from the readme:
```
    "Because JAX installation is different depending on your CUDA version, JMP does
     not list JAX as a dependency in `requirements.txt`."
```